src/harpstrata/IsActive/getActiveIds/getCounterpartActiveIds/reduceDisplayValueMatricesForActiveIds/index.ts

```diff
@@ -0,0 +1,3 @@
+export { reducePitchMatrixForActiveIds, reduceDegreeMatrixForActiveIds } from './reduceDisplayValueMatrixForActiveIds'
```

Comment:
These functions and files can be renamed to be shorter:

- folder: MatrixToActiveIdsReducers
- function: degreeMatrixReducer
- function: pitchMatrixReducer